client/app/src/views/userScreenshots.html

```diff
@@ -1 +1,11 @@
-Hello from userScreenshots.html
+<!-- Template for UserScreenshots -->
```

Comment:
I believe the userScreenshots will show that user's screenshots, public and private, versus the landing screenshots will be all users' public screenshots since no one has logged in yet..